plat/common/x86/syscall.S

```diff
@@ -65,54 +67,110 @@ ENTRY(_ukplat_syscall)
 	 * in the Red Zone.
 	 */
 	movq	%gs:LCPU_AUXSP_OFFSET, %rsp
+
 	/* Describing the rsp relative to GS would make it necessary to emit
 	 * raw CFI. Instead of doing so, mark rsp as undefined temporarily
 	 */
 	.cfi_undefined rsp
 
+	subq	$(UKARCH_AUXSPCB_SIZE), %rsp
+	movq	UKARCH_AUXSPCB_OFFSETOF_CURR_FP(%rsp), %rsp
+
+	/* We subtract 8 less bytes here so that we have room for the pushed
```

Comment:
-less

Reply:
That would be correct in fact.
```asm
	subq	$(UKARCH_EXECENV_END_ALIGN - 8), %rsp
	                                    ^^
	.cfi_adjust_cfa_offset (UKARCH_EXECENV_END_ALIGN - 8)

	pushq	%gs:LCPU_AUXSP_OFFSET <------------------------ the reason
	.cfi_adjust_cfa_offset 8

```

Reply:
Hum... is it then "we substract 8 bytes less"?